src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -42,6 +44,8 @@ fluent-toolbar[orientation=horizontal] {
     --error: #F8040A !important;
     --foreground-subtext-rest: #8D8D8D;
     --foreground-subtext-hover: #A1A1A1;
+    --error-ui-foreground-color: var(--foreground-on-accent-rest);
```

Comment:
`--foreground-on-accent-rest` doesn't seem right here... the background isn't the accent color, it's `--highlight-bg`. But then that also makes me wonder, if we're using a non-themed color for the background, shouldn't we use a non-themed color for the foreground (if we decided to tweak the luminosity later, the foreground color could change, but the background color wouldn't, so there'd be no contrast guarantee)?

Reply:
Yeah, I still wanted to base the colors off of existing themed ones, but you're right. I'm going to keep the same hex values, but put it explicitly and remove the reference.